### ISSUE TITLE
LgToggleComponent has an empty aria-describedBy attribute

### ISSUE THREAD

**Issue by thatguynamedandy** [Legal-and-General/canopy#56]
The toggle component has an empty aria-describedBy attribute, this is being flagged in WAVE as an a11y violation, AXE currently does not flag it.

https://legal-and-general-canopy.netlify.app/?path=/story/components-form-checkbox--standard

I had a quick scan but couldn't confirm 100% if an empty attribute is an error, but either way it's probably bad practice to have unnecessary attributes kicking around. This attribute is correctly populated if there is a validation error, but it should be omitted if there isn't an ID for it to point to.
